spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -337,6 +339,15 @@ public SparkContext createSparkContext() {
       conf.set("spark.yarn.isPython", "true");
     }
 
+    if (conf.contains("spark.yarn.keytab") && conf.contains("spark.yarn.principal")) {
+      try {
+        String keytab = conf.get("spark.yarn.keytab");
```

Comment:
This is not zeppelin keytab. It is just keytab specified by zeppelin. Every interpreter has its own setting, and this is just one of the setting.